README.md

```diff
@@ -3,3 +3,73 @@
 Smoker is a [smoke testing](https://en.wikipedia.org/wiki/Smoke_testing_%28software%29) tool aimed at verifying [Foreman](https://theforeman.org/)'s functionality and testing for regressions. Testing plug-ins is also within scope, but it should be optional.
 
 It's written in Python 3 and heavily relies on [pytest](https://pytest.org). To ensure it doesn't rely on anything on the system under test the ideal deployment is on a separate machine.
+
+# Running tests
+
+First install the dependencies:
+
+```sh
+pip install -r requirements.txt
+```
+
+To run, a base URL needs to be passed in:
+
+```
+pytest --base-url https://foreman.example.com
```

Comment:
How does this (or rather the driver) handle TLS certs? My test setups usually don't have valid certs deployed.

Reply:
By default it ignores all CA validation. If we want to verify that, it'd make sense to write more dedicated tests for that.

Reply:
Nah, I am good with not checking certs at all for now. thanks